.github/workflows/build.yml

```diff
@@ -244,7 +244,7 @@ jobs:
           move .\packaging\k6-$env:VERSION-windows-amd64\k6.exe .\packaging\
           rmdir .\packaging\k6-$env:VERSION-windows-amd64\
       - name: Add signtool to PATH
-        run: echo "${env:ProgramFiles(x86)}\Windows Kits\10\bin\x64" | Out-File -FilePath $env:GITHUB_PATH -Append
+        run: echo "${env:ProgramFiles(x86)}\Windows Kits\10\bin\10.0.17763.0\x64" | Out-File -FilePath $env:GITHUB_PATH -Append
```

Comment:
This looks really brittle :thinking:

Reply:
It is, but unfortunately in my search on the internet M$ hasn't made this anyway that this isn;'t like this that I can find. So luckily they only realease windows rarely.

Reply:
If you don't mind let's open the issue for fixing that in the long term 👍 but right now yeah, it's probably better to merge it as it's if it works